src/components/clusterDetail/ConsoleModal.tsx

```diff
@@ -30,39 +31,122 @@ type ConsoleModalProps = WebConsoleHintProps & {
   isOpen: boolean;
 };
 
+type ModalExpandableSectionProps = {
+  toggleText: string;
+  className: string;
+  isExpanded: boolean;
+  onToggle: () => void;
+  requiredList: string[];
+  optionalList: string[];
+};
+
+const formatMultilineString = (multiLineList: string[]): string =>
+  multiLineList && multiLineList.length > 0
+    ? [...multiLineList.slice(1)].reduce((accu, elem) => `${accu}\n${elem}`, multiLineList[0])
+    : '';
+
+const ModalExpandableSection: React.FC<ModalExpandableSectionProps> = (props) => {
+  const { requiredList, optionalList, ...otherProps } = props;
+
+  return (
+    <ExpandableSection {...otherProps}>
+      <TextContent>
+        <TextList>
+          <PrismCode code={formatMultilineString(requiredList)} language="yaml" />
```

Comment:
`markup` seems to work best for the plain configuration file

```suggestion
          <PrismCode code={formatMultilineString(requiredList)} language="markup" />
```